### PR TITLE
Preload updates

### PR DESCRIPTION
**Pull request by doanac**
Trying to put a little more content about scenarios that have been popping up lately.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 4649a4dd6af90be2984a58e4e755845174b13318 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/974/docs/artifacts/html/index.html

File: source/reference-manual/linux/preloaded-images.rst [hunk 1]
```diff
@@ -84,12 +94,44 @@ This can be configured in `factory-config.yml` with::
...
+have the same set of apps/containers. It's generally recommended
+to not produce a preloaded image. However, a ``ref_option`` could
+be added to set ``ASSEMBLE_SYSTEM_IMAGE: "0"`` for that branch.
+
+``APP_SHORTLIST`` will pick up its override value from the
+"refs/heads/sec-fix" ``ref_option``. If devel and master had
+different apps such as::
+
+  devel: fiotest,moneymaking-app,debug-tools
+  master: moneymaking-app
```
Comment (Tim-Anderson) [inline source/reference-manual/linux/preloaded-images.rst:132]: what if moneymaking-app is different in devel vs. master. Which would be used?

**Review by mike-sul — APPROVED** [review]
mind-blowing :)

File: source/reference-manual/linux/preloaded-images.rst (hunk 1 above)
Comment (mike-sul) [reply source/reference-manual/linux/preloaded-images.rst:132]: App is referenced by a unique URL that contains hash, the App URL is referenced by Target, each branch produces its own Target, the preload function preloads App that the current Target refers to.

Comment (doanac) [reply source/reference-manual/linux/preloaded-images.rst:132]: I could clarify this with better wording like "In this case the
"master" Target will have money-making-app preloaded from the container's master branch...."

does that help or just make it too "wordy"?

Comment (Tim-Anderson) [reply source/reference-manual/linux/preloaded-images.rst:132]: it would help some but why is it coming from master instead of devel. That is the question

Comment (doanac) [reply source/reference-manual/linux/preloaded-images.rst:132]: the master target gets containers from master. the devel target gets containers from devel. So the full sentence would be `In this case the "master" Target will have money-making-app preloaded from the container's master branch and the "devel" Target will have both money-making-app and debug-tools preloaded from the container's devel branch`

**Comment by Tim-Anderson** [comment]
LGTM

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**